PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/lib/appservice/AppServiceConfig.java

```diff
@@ -50,4 +57,16 @@
         result.put("pricingTier", Optional.ofNullable(pricingTier).map(PricingTier::getSize).orElse(StringUtils.EMPTY));
         return result;
     }
+
+    public static Region getDefaultRegion() {
+        final String environment = AzureEnvironmentUtils.azureEnvironmentToString(Azure.az(AzureAccount.class).account().getEnvironment());
```

Comment:
bad code to switch on azureEnvironmentToString(which is the display ui logic), use if (Azure.az(AzureAccount.class).account().getEnvironment() == AzureEnvironment.CHINA ?

Reply:
we have a list-region api, shall we use the first region of that list?